ansible_collections/arista/avd/roles/eos_validate_state/python_modules/tests/avdtestsecurity.py

```diff
@@ -36,3 +36,52 @@ def test_definition(self) -> dict | None:
         anta_tests.append({"VerifyAPIHttpsSSL": {"profile": profile, "result_overwrite": {"custom_field": f"eAPI HTTPS SSL Profile: {profile}"}}})
 
         return {self.anta_module: anta_tests}
+
+
+class AvdTestIPSecurity(AvdTestBase):
+    """
+    AvdTestIPSecurity class for IP security connection tests.
+    It validates the state of IPv4 security connections for a specified peer as established.
+    For now, it will only validate the peer under VRF default.
```

Comment:
```suggestion
    For now, it will only validate the peers under VRF default.
```

Reply:
so actually for now you only test IPsec pathgroup from router path selection so we need to mention this in doc and in the docstring as a limitation.
You would not support other way of deifning ipsec tunnels today